Scalar.Common/ScalarPlatform.cs

```diff
@@ -114,6 +114,37 @@ public static void Register(ScalarPlatform platform)
             PhysicalFileSystem fileSystem,
             ITracer tracer);
 
+        public bool TryGetCredentialStore(ITracer tracer, PhysicalFileSystem fileSystem, out ICredentialStore credentialStore, out string errorMessage)
+        {
+            credentialStore = null;
+
+            string gitBinPath = ScalarPlatform.Instance.GitInstallation.GetInstalledGitBinPath();
```

Comment:
We still always prefer to use Git and `git-credential` over any bundled GCM, that way we benefit from the likely more up-to-date version of GCM.

We don't expect all distributions of Scalar to have GCM bundled, just scenarios that need to be self-installing/bootstrapping. If GCM is not bundled in the distribution, the existing behaviour is maintained (fail with error "cannot find Git").